packages/fxa-auth-server/lib/senders/emails/partials/paymentPlanDetails/en.ftl

```diff
@@ -0,0 +1,4 @@
+payment-plan-product-name = { $productName }
```

Comment:
We shouldn't need an ID here when it's just for something that won't be translated.

Also, this file is missing those variable comments.